### PR TITLE
addWidget() lose all config data

### PR DESCRIPTION
**Pull request by liutsi**
addWidget() lose all config data, pipeline first-time save fails.

### CODE REVIEW

**Comment by rvema** [comment]
@liutsi  There is a separate effort from @Sbrenthughes , Can you you reach out to him regarding this?

**Comment by Sbrenthughes** [comment]
Looks like a different issue @rvema

File: api/src/main/java/com/capitalone/dashboard/service/DashboardRemoteServiceImpl.java [hunk 1]
```diff
@@ -189,6 +189,11 @@ private Dashboard requestToDashboard(DashboardRemoteRequest request) throws Hygi
             serviceName = service.getConfigurationItem();
         }
         List<String> activeWidgets = new ArrayList<>();
-        return new Dashboard(true, metaData.getTemplate(), metaData.getTitle(), application, metaData.getOwner(), DashboardType.fromString(metaData.getType()), serviceName, appName,activeWidgets, false, ScoreDisplayType.HEADER);
+
+        Dashboard dashboard = new Dashboard(true, metaData.getTemplate(), metaData.getTitle(), application, metaData.getOwner(), DashboardType.fromString(metaData.getType()), serviceName, appName,activeWidgets, false, ScoreDisplayType.HEADER);
+
+        //remoteCreated is actually ignored in Dashboard construction method, so set it again.
+        dashboard.setRemoteCreated(true);
```
Comment (Sbrenthughes) [inline api/src/main/java/com/capitalone/dashboard/service/DashboardRemoteServiceImpl.java]: This is the incorrect place to make this change. It should probably be fixed in the Dashboard.java class as your comment suggest.

Comment (rvema) [reply api/src/main/java/com/capitalone/dashboard/service/DashboardRemoteServiceImpl.java]: @liutsi Can you look into the comments above and update this PR?

Comment (rvema) [reply api/src/main/java/com/capitalone/dashboard/service/DashboardRemoteServiceImpl.java]: @liutsi can you look into the above PR comments?

Comment (rvema) [reply api/src/main/java/com/capitalone/dashboard/service/DashboardRemoteServiceImpl.java]: @liutsi we will end up closing this PR due to inactivity, If you plan to contribute please create a new PR

**Comment by rvema** [comment]
Closing the PR as the contributor is inactive

**Pull request closed without merging**